calico-cloud/users/user-management.mdx

```diff
@@ -40,7 +40,7 @@ The Owner role cannot be assigned to new users. The only Owner is the user who c
 
 {{prodname}} supports the following IdPs:
 
-- OIDC-based Auth providers (for example Google, Azure AD)
+- Any OIDC-based Auth providers (for example, OKTA, Google, Azure AD)
```

Comment:
Suggestion:

{{prodname}} works with any identity provider that supports [OpenID Connect](https://openid.net/connect/).

To add an identity provider, open a [Support ticket](https://support.tigera.io/).

Reply:
I'm also in favor of s/IdP/identity provider in all instances.

Reply:
Agree. Fixed.

Reply:
Fixed in CC.